lib/src/searchfield.dart

```diff
@@ -7,6 +7,17 @@ enum SuggestionState {
 
   /// hide the suggestions on initial focus
   hidden,
+
+  /// show suggestions only on tap
+  onTap,
+}
+
+enum SuggestionAction {
```

Comment:
We should have a description for this enum, Which explains what this enum is supposed to do.
something like " enum which defines the state of focus of the Searchfield when the suggestion is tapped"